### PR TITLE
Initial plumbing for Cloud 8 GM / Cloud9 development

### PR DESCRIPTION
**Pull request by dirkmueller**
susecloud8 -> susecloud9, and update references for GM8/GM8+up

### CODE REVIEW

**Comment by rsalevsky** [comment]
Travis failed.

**Comment by aspiers** [comment]
Please squash this fixup prior to merging, thanks!

**Comment by dirkmueller** [comment]
travis was fixed by @bmwiedemann - thanks !

File: scripts/qa_crowbarsetup-test.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 describe "roundup(1) testing of qa_crowbarsetup.sh"
 
-verlist="3 4 5 5minus 8 4plus 8M4plus 8M4minus"
+verlist="3 4 5 5minus 9 4plus 9M4plus 9M4minus"
```
Comment (aplanas) [inline scripts/qa_crowbarsetup-test.sh]: are 3, 4, 5, 5minus and 4plus version of cloud? if so can we remove them?

File: scripts/qa_crowbarsetup-test.sh [hunk 2]
```diff
@@ -41,7 +41,7 @@ getcloudversionmatrixrow() {
 
 it_returns_correct_getcloudver() {
     results=`getcloudversionmatrixrow develcloud3 GM3 develcloud4 GM4 develcloud5 susecloud5 GM5 M3 M4 M5 Beta3 Beta4 RC3 GMC`
-    test "$results" = "33445558888888"
+    test "$results" = "33445559999999"
```
Comment (aplanas) [inline scripts/qa_crowbarsetup-test.sh]: same here and in line 28

File: scripts/qa_crowbarsetup-test.sh (hunk 1 above)
Comment (bmwiedemann) [reply scripts/qa_crowbarsetup-test.sh]: They meant SOC3/4/5 but are used here only for testing the cloudver comparison (< = >) checks and should not be updated for every new version

**Pull request merged**